tests/validation/tests/v3_api/common.py

```diff
@@ -1031,16 +1052,8 @@ def validate_dns_entry(pod, host, expected, port=TEST_IMAGE_PORT, retry_count=3)
     if HARDENED_CLUSTER:
```

Comment:
since this is here to determine `cmd`, shouldn't you remove the check for hardened cluster in the `retry_cmd` function?